src/test/kotlin/no/nav/syfo/sykmeldingstatus/SendSykmeldingEndeTilEndeSpek.kt

```diff
@@ -0,0 +1,153 @@
+package no.nav.syfo.sykmeldingstatus
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.application.install
+import io.ktor.features.ContentNegotiation
+import io.ktor.http.ContentType
+import io.ktor.http.HttpMethod
+import io.ktor.jackson.jackson
+import io.ktor.routing.routing
+import io.ktor.server.testing.TestApplicationEngine
+import io.ktor.server.testing.handleRequest
+import io.ktor.server.testing.setBody
+import java.sql.ResultSet
+import java.time.LocalDateTime
+import no.nav.syfo.db.DatabaseInterface
+import no.nav.syfo.db.toList
+import no.nav.syfo.objectMapper
+import no.nav.syfo.persistering.opprettBehandlingsutfall
+import no.nav.syfo.persistering.opprettSykmeldingsdokument
+import no.nav.syfo.persistering.opprettSykmeldingsopplysninger
+import no.nav.syfo.sykmeldingstatus.api.ArbeidsgiverDTO
+import no.nav.syfo.sykmeldingstatus.api.ShortNameDTO
+import no.nav.syfo.sykmeldingstatus.api.SporsmalOgSvarDTO
+import no.nav.syfo.sykmeldingstatus.api.SvartypeDTO
+import no.nav.syfo.sykmeldingstatus.api.SykmeldingSendEventDTO
+import no.nav.syfo.sykmeldingstatus.api.registerSykmeldingSendApi
+import no.nav.syfo.testutil.TestDB
+import no.nav.syfo.testutil.dropData
+import no.nav.syfo.testutil.testBehandlingsutfall
+import no.nav.syfo.testutil.testSykmeldingsdokument
+import no.nav.syfo.testutil.testSykmeldingsopplysninger
+import org.amshove.kluent.shouldEqual
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class SendSykmeldingEndeTilEndeSpek : Spek({
```

Comment:
Når man tester alle lag for seg selv, API, Service og Database så er det ikke sikkert man trenger en ende til ende test.

Reply:
Egentlig synes jeg ende-til-ende-testen gir mer verdi, hvertfall når det ikke er noe særlig logikk i hvert lag, så da vil jeg heller fjerne de andre :)